examples/gcp_cli.py

```diff
@@ -102,6 +102,24 @@ def QueryLogs(args: 'argparse.Namespace') -> None:
     print(json.dumps(line))
 
 
+def CreateDiskFromGCSImage(args: 'argparse.Namespace') -> None:
+  """Creates GCE persistent disk from image in GCS.
+
+  Args:
+      args (argparse.Namespace): Arguments from ArgumentParser.
+  """
```

Comment:
Please point to the docstring for `CreateDiskFromGCSImage` to get the details about how the image is created.

Reply:
Done